src/components/site/Accordion.md

```diff
@@ -38,15 +38,13 @@ const collapseItems = [
     header: "Accordion Item #4",
     subheader: "",
     content: (
-      <img
-        width="300"
-        height="200"
-        alt="Stuff goes here"
-        src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
-        border="0"
-        vspace="0"
-        hspace="0"
-      />
+      <div id="dg_main-content">
```

Comment:
Note that in this case, you are repeating the same id `dg_main-content` multiples times which is invalid. There can only be one id per page, this will generate multiples. I'm assuming you need the styles associated with 'dg_main content'?

Reply:
per our chat, this id can be duplicated in the components until we come up for a solution for this problem.